client/verta/docs/tutorials/deployment/endpoint_creation.rst

```diff
@@ -0,0 +1,35 @@
+Creating and Retrieving an Endpoint using the Client and CLI
+============================================================
+
+Verta's Endpoint provides an API for configurable deployment of machine learning models.
```

Comment:
This is close, but maybe a little more context. I'd recommend phrasing it in a way where instead of describing _what_ an Endpoint is, we start with a user story for _why_ they would use this.

Something like
>When a machine learning model is released, it must be made available for downstream consumption; Verta offers Endpoints as an API for configuring and releasing deployable models.

Reply:
Yeah that provides more context. Sg. Fixed.